README.md

```diff
@@ -64,3 +63,12 @@ pip install -r requirements.txt
 ```
 python3 tests/runtests.py
 ```
+
+### Deploying
+```
+pip-compile --no-emit-index-url requirements-build.in
```

Comment:
Is this really needed every time? This should only be done if we're updating build requirement packages. I think we should remove this from the deploy steps.

Reply:
Updated everywhere, this look better to you?